pkgs/development/python-modules/python-dbusmock/default.nix

```diff
@@ -8,9 +8,15 @@
 , bluez
 , networkmanager
 , setuptools-scm
+, runCommand
 }:
 
-buildPythonPackage rec {
+let
+  pbap-client = runCommand "pbap-client" { } ''
+    mkdir -p "$out/bin"
+    ln -s "${bluez.test}/test/pbap-client" "$out/bin/pbap-client"
+  '';
```

Comment:
How about we just append ``${bluez.test}/test`` to PATH in preCheck? Thats faster and cleaner.

Reply:
Sure, I can do that

Reply:
Wait, now I remember why we can't do that. String substitution like that is constructed whether or not checkPhase is actually used. that unconditional inclusion introduces a dependency cycle when trying to add pcsclite as a dependency of libfido2.

Reply:
Forgot about that too, maybe we should add a comment about that so that we do not try it again in the future. Something like “Cannot just add it to path in `preCheck` since that attribute will be passed to `mkDerivation` even with `doCheck = false`, causing a dependency cycle.”

Reply:
Good idea.